collect_app/src/main/java/org/odk/collect/android/utilities/CameraUtils.java

```diff
@@ -0,0 +1,51 @@
+package org.odk.collect.android.utilities;
+
+import android.annotation.TargetApi;
+import android.hardware.Camera;
+import android.hardware.camera2.CameraAccessException;
+import android.hardware.camera2.CameraCharacteristics;
+import android.hardware.camera2.CameraManager;
+import android.os.Build;
+
+import org.odk.collect.android.application.Collect;
+
+import timber.log.Timber;
+
+/**
+ * Created by Akshay on 10/11/17.
+ */
+@TargetApi(21)
```

Comment:
You don't need this.

Reply:
If `@TargetApi(21)` is removed, I get error during ` /gradlew lint -Pandroid.useDexArchive=false
 ` task execution as `Class requires API level 21 (current min is 16): android.hardware.camera2.CameraAccessException`. Is there any other way to do this?